### ISSUE TITLE
/usr/src/app/llamafile-main: 1: Not: not found

### ISSUE THREAD

**Issue by Aviksaikat** [iverly/llamafile-docker#2]
Hi, I'm using the [llava-v1.5-7b-Q8_0.gguf](https://huggingface.co/jartine/llava-v1.5-7B-GGUF/blob/main/llava-v1.5-7b-Q8_0.gguf) model & ran this command  

```sh
docker run -it --rm \                   
-v llava-v1.5-7b-Q8_0.gguf:/model \
iverly/llamafile-docker:main-latest -m /model
```
and this is the error I got. Same with `llamafile-server`

```sh
Status: Downloaded newer image for iverly/llamafile-docker:main-latest
/usr/src/app/llamafile-main: 1: Not: not found
```

# System
```sh
Linux kali 6.5.0-kali3-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.5.6-1kali1 (2023-10-09) x86_64 GNU/Linux
```

**Comment by iverly** [comment]
Hello,

This is due to changes made in the latest version (v0.5).

I will come up with a fix in the next hour.
If you want to use it now, you can downgrade to (v0.4).

**Comment by iverly** [comment]
Fixed in [8123b74](https://github.com/iverly/llamafile-docker/commit/8123b7429e7753543ba56c57d3a268928679b4d2).
